src/opm/parser/eclipse/share/keywords/900_OPM/S/SPIDERWEB

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "SPIDERWEB",
```

Comment:
Maybe we should limit ourselves to 8 characters?

Reply:
Ok, will switch to SPIDER.

Reply:
That is what the manual says: "Cell dimensions, permeabilities and transmissibilities may then be specified using radial
alias keywords, and are interpreted as being in the _radial geometry coordinate system_." From all the tests we have run, we have no reason to believe is otherwise.

Reply:
> From all the tests we have run, we found no reason to believe is otherwise.

OK - with some help from Equinor I have done a test myself; I agree that a proper cylindrical coordinate system is used with the `RADIAL`grid.